### DISCUSSION TITLE
Browser Support

### DISCUSSION THREAD

**Discussion by risadams** [selectize/selectize.js#1782] [Q&A]
In preparation for the future, If you use selectize in a project, wheat browsers do you target?

Specifically, it's probably time to drop IE support, but we want to make sure that we don't leave people stranded on older versions.

**Comment by databyte** [comment]
Most libraries have dropped support for IE11 and simply list the last supported version in their README for users to lock/pin for compatibility.

**Comment by bago** [comment]
I just selected selectize.js for my project and I still need IE11 support, but I can stick to older releases.
